examples/src/bin/rtc_time.rs

```diff
@@ -5,6 +5,8 @@
 #![no_std]
 #![no_main]
 
+use core::time::Duration;
```

Comment:
Ah, I guess this is one reason not to use `chrono`, as `core::time::Duration` is quite overkill for embedded systems, but we can address this later if needed.

Reply:
Would `chrono::TimeDelta` be any better? It's pretty much the same as `core::time::Duration` though.

Reply:
No this is fine as is, more of an observation for future me :D.